shared/src/main/java/uk/gov/di/authentication/shared/helpers/PhoneNumberHelper.java

```diff
@@ -20,6 +20,16 @@ public static String formatPhoneNumber(String phoneNumber) {
         }
     }
 
+    public static String getCountry(String phoneNumber) {
+        try {
+            return Integer.toString(
```

Comment:
isn't country code a string?

Reply:
Unfortunately not, `getCountryCode()` returns an int